apps/admin-ui/src/spa/recurring-reservations/application-rounds/[id]/review/RejectedOccurrencesDataLoader.tsx

```diff
@@ -0,0 +1,195 @@
+import { useNotification } from "@/context/NotificationContext";
+import { useSort } from "@/hooks/useSort";
+import { SORT_KEYS } from "./AllocatedEventsTable";
```

Comment:
This looks wrong. If they are the same I'd rather re-export from `RejectedOccurrencesTable` if they aren't then fix the export.

Looks like a incorrect automatic export (since `RejectedOccurrencesTable` exports `SORT_KEYS`)

Reply:
Yup, it's an artifact left from an incorrect auto-import. Should be imported from `RejectedOccurrencesTable`.